### DISCUSSION TITLE
Running actual version on Linux

### DISCUSSION THREAD

**Discussion by flokixdev** [gitextensions/gitextensions#8143] [General] [answered]
Hi, regarding this article https://github.com/gitextensions/gitextensions/wiki/How-To:-run-Git-Extensions-on-Linux I should download GitExtensions-2.51.05-Mono.zip if I want to run GitExt on Linux.

Is it possible to download an actual version? E.g. v3.3.1?

**Comment by RussKie** [comment, marked as answer]
No, v3.x+ are Windows only
https://github.com/gitextensions/gitextensions/wiki/Compare-versions

**Reply by flokixdev** [reply]
It's a pity, thanks for the clarification.

**Comment by gerhardol** [comment]
See #6051 for attempts to run GE 3 in wine and #6239 for a path to native support.
